src/DssDirectDepositAaveDai.t.sol

```diff
@@ -328,6 +328,24 @@ contract DssDirectDepositAaveDaiTest is DSTest {
         assertEq(vat.dai(address(deposit)), 0);
     }
 
+    function test_bar_zero() public {
+        uint256 targetBorrowRate = _setRelBorrowTarget(7500);
+        deposit.reap();     // Clear out interest to get rid of rounding errors
+        assertEqInterest(getBorrowRate(), targetBorrowRate);
+
+        (uint256 ink, uint256 art) = vat.urns(ilk, address(deposit));
+        assertGt(ink, 0);
+        assertGt(art, 0);
```

Comment:
Why does the setup of this test result in `ink` and `art` = 0 when this test end up with some `ink` and `art`?
https://github.com/makerdao/dss-direct-deposit/blob/716821f80264158c193602ec70ea746a6ed4fd65/src/DssDirectDepositAaveDai.t.sol#L300-L308

Reply:
It's a greater than. :) I miss those sometimes too.